.isort.cfg

```diff
@@ -0,0 +1,5 @@
+[isort]
+multi_line_output=4
+not_skip = __init__.py
+skip = .tox
+recursive = true
```

Comment:
Isort is a module import order sorter. It can do its magic automatically, but it is always good to have it as a CI step, to improve code beauty.